sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -82,6 +83,7 @@ class AdaptiveQueryExecSuite extends QueryTest with SharedSQLContext {
   test("Change merge join to broadcast join") {
     withSQLConf(
         SQLConf.RUNTIME_REOPTIMIZATION_ENABLED.key -> "true",
+        SQLConf.REDUCE_POST_SHUFFLE_PARTITIONS_ENABLED.key -> "false",
```

Comment:
Any reason for this?

Reply:
I added the same test with `REDUCE_POST_SHUFFLE_PARTITIONS_ENABLED` set to true and checked the post shuffle partition. So I set it to false in this test.

Reply:
I don't think it's necessary, but if you do wanna test it with "false", how about do a combo: `Seq("true", "false").foreach`?

Reply:
Ok, let me just remove this for now.